lexik/jwt-authentication-bundle/2.3/Makefile

```diff
@@ -0,0 +1,5 @@
+generate-rsa-keys:
```

Comment:
`generate-jwt-keys` could be a better name? `RSA` looks a bit generic, but `JWT` should be clear enough.